Documentation/Books/Manual/Administration/Configuration/README.md

```diff
@@ -0,0 +1,286 @@
+# Configuration
+
+The [programs and tools](../../Programs/README.md) shipped in an
+ArangoDB package can be configured with various _startup options_.
+
+- Startup options you specify on a command line are referred to as
+  [command line options](#command-line-options):
+
+  `arangosh --server.database myDB`
+
+- The same options can also be set via
+  [configuration files](#configuration-file-format),
+  using a slightly different syntax:
+
+  `server.database = myDB`
+
+- There are also _flags_ which are for command line usage only,
+  such as `‑‑help` and `‑‑version`. They don't take any value
+  in contrast to options.
+
+Find the available options and flags in the _Options_ sub-chapters of the
+respective [Programs & Tools](../../Programs/README.md) sub-chapter, like the
+[ArangoDB Server Options](../../Programs/Arangod/Options.md).
+
+The [ArangoDB Starter](../../Programs/Starter/README.md) works differently
+to the other programs and tools. It uses `setup.json` files for its own
+[configuration](../../Programs/Starter/Architecture.md#starter-data-directory)
+and has a fluent command line interface to execute certain actions.
+If you deploy ArangoDB with the Starter, then custom `arangod.conf` files
+are generated by this tool and are used instead of the default configuration.
+
+## Command line options
+
+Command line options can be supplied in the style `‑‑option value` with two
+dashes (also known as hyphen minus), the name of the option, a space as
+separator and the value. You may also use an equals sign `=` as separator
+like `‑‑option=value`.
+
+The value can be surrounded with double quote marks `"` like
+`‑‑option="value"`. This is mandatory if the value contains spaces,
+but it is optional otherwise.
+
+Some binaries accept one unnamed argument, which means you can take a
+shortcut and leave out the `‑‑option` part and supply the value directly.
+It does not matter if you supply it as first or last argument, or between
+any of the named arguments. For _arangod_ it is the `‑‑database.directory`
+option. The following commands are identical:
+
+```
+arangod my_data_dir
+arangod "my_data_dir"
+arangod --database.directory my_data_dir
+arangod --database.directory=my_data_dir
+arangod --database.directory "my_data_dir"
+arangod --database.directory="my_data_dir"
+```
+
+Many options belong to a section as in `‑‑section.param`, e.g.
+`‑‑server.database`, but there can also be options without any section.
+These options are referred to as _global options_.
```

Comment:
I am unsure about this. why global option?  this may be confusing. I would call global option also options like --database.directory, e.g. options that can be set globally but not "per session" (e.g. you set the data dir globally and can not be changed per session). other options can be global or also set locally (e.g. some query options). in this case the local value is used.  I would just removed "these options are reffered to as _global options_. WDYT

Reply:
I did not make up this name, there is a section called "global options" if you run any of the binaries with `--help`, which lists all options without a dot in their name. This is a bit misleading:

```
>arangosh --help
Usage: arangosh [<options>]

Section 'global options' (Global configuration)
  --configuration <string>                          the configuration file or 'none' (default: "")
  --jslint <string...>                              do not start as shell, run jslint instead (default: )
  --quiet <boolean>                                 silent startup (default: false)
  --version <boolean>                               reports the version and exits (default: false)

Section 'console' (Configure the console)
  --console.audit-file <string>                     audit log file to save commands and results (default: "")
  --console.auto-complete <boolean>                 enable auto completion (default: true)
...
```

If you want to change this, please propose a new name and ask @jsteemann to change it in the code:

- https://github.com/arangodb/arangodb/blob/devel/lib/ApplicationFeatures/ApplicationServer.cpp#L308
- https://github.com/arangodb/arangodb/blob/devel/lib/ProgramOptions/Option.cpp#L124
- elsewhere?

Reply:
thanks for the clarification!